string robot_id
int16 status
geometry_msgs/Point position